lib/spear/connection/request.ex

```diff
@@ -216,12 +222,12 @@ defmodule Spear.Connection.Request do
 
   def handle_data(%__MODULE__{type: {:on_data, on_data}} = request, new_data) do
     case Spear.Grpc.decode_next_message(request.response.data <> new_data, request.response.type) do
-      {%ReadResp{content: {:confirmation, %{subscription_id: _}}}, rest} ->
+      {read_resp(content: {:confirmation, _confirmation}), rest} ->
```

Comment:
as you can see here, the `Record.defrecord/2` produced macros made it really easy to convert from the structs of `elixir-protobuf/protobuf` to gpb records